design/design-doc-stacks.md

```diff
@@ -238,10 +238,11 @@ In this example, the directory names "databases.foocompany.io", "mysql", "v1alph
 * `icon.svg`: This file (or `icon.png`, `icon.jpg`, `icon.gif`, or potentially other supported filenames, TBD) will be used in a visual context when listing or describing this stack.  The preferred formats/filenames are `svg`, `png`, `jpg`, `gif` in that order (if multiple files exist).  For bitmap formats, the width to height ratio should be 1:1. Limitations may be placed on the acceptable file dimensions and byte size (TBD).
 * `resources` directory: This directory contains all the CRDs and optional metadata about them.
   * `group.yaml`: Related Stack resources (CRDs) can be described at a high level within a group directory using this file.
-  * `*resource.yaml` and `icon.svg`: Files that describe the resource with descriptions, titles, or images, may be used to inform out-of-cluster or in-cluster Stack managing tools.  This may affect the annotations of the `Stack` record or the Resource CRD (TBD).
-  Multiple `*resource.yaml` files may exist alongside multiple CRD files in the same directory.  The `resource.yaml` files should include an `id:` referencing the `Kind` of their matching CRD.
+  * `*resource.yaml` and `*icon.svg`: Files that describe the resource with descriptions, titles, or images, may be used to inform out-of-cluster or in-cluster Stack managing tools.  This may affect the annotations of the `Stack` record or the Resource CRD (TBD).
+    * Multiple `*resource.yaml` files may exist alongside multiple CRD files in the same directory.  The `resource.yaml` files should include an `id:` referencing the `Kind` of their matching CRD.
+    * Multiple `*icon.svg` files may be included in the same directory as the CRD yaml files they modify. In this case the filename should be prefixed to match the `Kind` of the associated CRD (`mytype.icon.svg`).
   * `*ui-schema.yaml`: UI metadata that will be transformed and annotated according to the [Stack UI Metadata One Pager](one-pager-stack-ui-metadata.md)
-  Multiple `ui-schema.yaml` files may be included in the same directory as the CRD yaml files they modify. In this case the filename should be prefixed to match the `Kind` of the associated CRD (`mytype.ui-schame.yaml`).
+    * Multiple `*ui-schema.yaml` files may be included in the same directory as the CRD yaml files they modify. In this case the filename should be prefixed to match the `Kind` of the associated CRD (`mytype.ui-schema.yaml`).
```

Comment:
Nice fix! It would have been a `schame` to not have caught this before releasing!